scripts/update/Updater.php

```diff
@@ -715,5 +715,35 @@ public function update($initialVersion) {
 
             $this->setVersion('5.31.0');
         }
+        
+        if ($this->isVersion('5.31.0')) {
+
+            $extension = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiTest');
+
+            $config = $extension->getConfig('testRunner');
+            $config['shortcuts'] = [
+                'calculator' => [
+                    'toggle' => 'C',
+                ],
+                'zoom' => [
+                    'in' => 'I',
+                    'out' => 'O'
+                ],
+                'comment' => [
+                    'toggle' => 'A',
+                ],
+                'itemThemeSwitcher' => [
+                    'toggle' => 'T',
+                ],
+                'review' => [
+                    'toggle' => 'R',
+                    'flag' => 'M'
+                ]
+            ];
+
```

Comment:
I think this is the same problem with the plugins' config.
The script that updates the config has to be clever enough to avoid unwanted overwriting.

For instance:
```php
// keep other existing shortcuts
$testConfig['shortcuts']['calculator']['toggle'] = 'C';
```

Instead of:
```php
// loose existing shortcuts
$testConfig['shortcuts'] = [
    'calculator' => [
        'toggle' => 'C'
    ]
];
```

Or even:
```php
// still loose existing shortcuts
$testConfig = array_merge($testConfig, [
    'shortcuts' => [
        'calculator' => [
            'toggle' => 'C'
        ]
    ]
]);
```